python/sdssdb/etc/sdssdb.yml

```diff
@@ -51,3 +51,9 @@ utahdb:
     host: db.sdss.utah.edu
     port: 5432
     domain: db.sdss.utah.edu
+
+slore:
```

Comment:
Why slore?

Reply:
I needed a profile for the `sdss` user on the `lore` host machine.  I already had a `lore` profile for the read-only `marvin` database user which is only relevant for the `manga` db. I wasn't really sure what to call this.  And what's the policy here on what actually goes in this file?  Are we supposed to put one new profile per host machine?  Or one new profile per database user per host machine?